MantidQt/API/src/ManageUserDirectories.cpp

```diff
@@ -119,6 +124,21 @@ void ManageUserDirectories::saveProperties() {
   config.saveConfig(m_userPropFile.toStdString());
 }
 
+/**
+ * Appends a forward slash to the end of a path if there is no slash (forward or
+ * back) there already, and strip whitespace from the path.
+ *
+ * @param path :: A reference to the path
+*/
+void ManageUserDirectories::appendSlashIfNone(QString &path) const {
+  path = path.trimmed();
+  if (path.endsWith("/") || path.endsWith("\\") || path.isEmpty()) {
```

Comment:
Is it possibly better to use:
```c++
if(!(path.endsWith("/") || path.endsWith("\\") || path.isEmpty())) { 
  path = path.append("/");
}
```
Rather than if-else with a return statement (i.e. using a not operator).